build.sh

```diff
@@ -74,9 +80,13 @@ function printHelp() {
     {
         printf " \t%s\t%s\n" "--test" "to run the tests"
         printf " \t%s\t%s\n" "--testUi" "to run the tests including UI tests"
+        printf " \t%s\t%s\n" "--installCore" "install JMC core"
         printf " \t%s\t%s\n" "--packageJmc" "to package JMC"
+        printf " \t%s\t%s\n" "--packageAgent" "to package Agent"
         printf " \t%s\t%s\n" "--clean" "to run maven clean"
         printf " \t%s\t%s\n" "--run" "to run JMC once it was packaged"
+        printf " \t%s\t%s\n" "--runAgentExample" "to run Agent 'InstrumentMe' example once it was packaged"
+        printf " \t%s\t%s\n" "--runAgentConverterExample" "to run Agent Converter 'InstrumentMeConverter' example, once Agent was packaged"
```

Comment:
Skip converter and the comma, to keep with the pattern for the other "run"->
 "--runAgentConverterExample" "to run Agent 'InstrumentMeConverter' example once it was packaged"